lib/webpacker/test_helper.rb

```diff
@@ -0,0 +1,25 @@
+require "rake"
+require "webpacker"
+
+module Webpacker::TestHelper
+  extend ActiveSupport::Concern
+
+  included do
+    setup :compile_webpack_assets
+  end
+
+  def compile_webpack_assets
+    Rails.cache.fetch(["webpacker", "manifest", checksum]) do
```

Comment:
@javan Yepp that's right. Good point 👍  Perhaps we should watch `package.json` and `yarn.lock` for change too? The compile task slows down the tests so it's best to cache because we won't be able to get news digests from webpack until it's compiled (which is same thing as re-compiling) unless I am missing something??